common/src/main/java/io/seata/common/DefaultValues.java

```diff
@@ -54,6 +54,8 @@
     boolean DEFAULT_TRANSACTION_UNDO_DATA_VALIDATION = true;
     String DEFAULT_TRANSACTION_UNDO_LOG_SERIALIZATION = "jackson";
     boolean DEFAULT_ONLY_CARE_UPDATE_COLUMNS = true;
+    boolean DEFAULT_DO_NOT_CARE_ON_UPDATE_TIMESTAMP_WHEN_INSERT = true;
```

Comment:
I think the default value change to false will be better when set up a new configuration.